STUDIO/org.openl.rules.mail/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.openl.rules</groupId>
+        <artifactId>org.openl.rules.studio-pom</artifactId>
+        <version>5.26.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <name>OpenL - STUDIO - Mail</name>
+    <description>OpenL Mail implementation</description>
+
+    <artifactId>org.openl.rules.mail</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context-support</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.velocity</groupId>
+            <artifactId>velocity</artifactId>

```

Comment:
The template is so simple that it can be replaced by

        try (var stream = getClass().getResourceAsStream("template")) {
            var template = IOUtils.toStringAndClose(stream);
            var message = template.replace("${link}", link);
        } catch (IOException e) {
            e.printStackTrace();
        }